js/app.js

```diff
@@ -138,16 +164,39 @@ function properCase(nameString) { // borrowed solution @ https://stackoverflow.c
 }
 
 // function to update the sales report if a new store is added via the HTML form
-function reportUpdate() {
+function updateNewStore() {
```

Comment:
Can you see similarities between this function and the second half of your updateExistingStore function?

Reply:
I suppose I could transition updateNewStore() to nameTbd() where I pass the index value to this function and just re-render the entire report body regardless of an existing store or new store (instead of only adding the new row for the new store. Then I could chop out the last half of updateExistingStore() and instead call that revised function.

The need to re-render the entire report for an existing store vs the minimum need to add a single row for a new store blocked me from seeing this possibility. That would save at least a a few lines of code. Thanks for the idea!